### DISCUSSION TITLE
how to add either border or slightly change the background of the terminal

### DISCUSSION THREAD

**Discussion by Dmitry-N-Medvedev** [LazyVim/LazyVim#734] [Q&A]
good morning,

when invoked the terminal window is not separated from the background.
I would love to either add borders to it, or to change its background color slightly.

PS: sorry for super newbie question.

best regards,
Dmitry

**Comment by dpetka2001** [comment]
If you use toggleterm like suggested in the other topic, it has an option `shade_terminals` that you can use to make the background darker than your colorscheme.
